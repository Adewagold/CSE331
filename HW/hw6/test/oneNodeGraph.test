# Tests build oneNodeGraph.

# Load oneNodeGraph.tsv
LoadGraph one_node_graph oneNodeGraph.tsv

# Print the nodes in the graph and the outgoing edges from onlyNode
ListNodes one_node_graph
ListChildren one_node_graph onlyNode