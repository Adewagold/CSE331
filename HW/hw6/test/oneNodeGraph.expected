# Tests build oneNodeGraph.

# Load oneNodeGraph.tsv
loaded graph one_node_graph

# Print the nodes in the graph and the outgoing edges from onlyNode
one_node_graph contains: onlyNode
the children of onlyNode in one_node_graph are: